packages/charts/src/charts/bubble/correlation.component.tsx

```diff
@@ -1,28 +1,53 @@
 import React, { FC } from 'react';
 import { Text, Typography } from '@keen.io/ui-core';
 
-import { Header, Container, Value, Point } from './correlation.styles';
+import {
+  Header,
+  Container,
+  Value,
+  Point,
+  SizeContainer,
+} from './correlation.styles';
 
 type Props = {
   typography: Typography;
   correlations: { name: string; value: number | string }[];
   title: string;
   color: string;
+  valueKey?: string;
```

Comment:
The `valueKey` is required in `BubbleChart` interface. Why it's optional here and initialized to empty `string` ?